package.json

```diff
@@ -25,16 +25,20 @@
     "@babel/plugin-proposal-class-properties": "^7.10.4",
     "@babel/preset-react": "^7.10.4",
     "babel-loader": "^8.1.0",
+    "nodemon": "^2.0.4",
     "webpack": "^4.44.0",
     "webpack-cli": "^3.3.12"
   },
   "dependencies": {
     "axios": "^0.19.2",
     "bcrpyt": "^2.0.0",
     "body-parser": "^1.19.0",
+    "bootstrap": "^4.5.0",
     "cookie-parser": "^1.4.5",
     "dotenv": "^8.2.0",
     "express": "^4.17.1",
+    "jquery": "^3.5.1",
```

Comment:
?

Reply:
i was considering using jquery in the future for some experimental stuff but that wont be until much further down the line